desk/app/channels.hoon

```diff
@@ -1136,17 +1140,19 @@
   ?>  |(from-self =(ask our.bowl))
   =/  [=wire =dude:gall =path]
     =/  base=path  (said-path nest plan)
-    ::  v3 subscriptions will _always_ ask the client agent,
+    ::  Old (v1, v2, v3) subscriptions will hit channels-server, newer versions
+    ::  may hit the client agent, because they want to hit the logic
+    ::  that circumvents channel permissions for pinned posts
     ::  because they want to hit the logic that circumvents channel permissions
     ::  for pinned posts
     ::
-    ?:  &(=(ask ship.nest) !?=(?(%v3 %v4) ver))
+    ?:  &(=(ask ship.nest) ?=(?(%v0 %v1 %v2 %v3) ver))
       [base server base]
     ::NOTE  attention! we subscribe to other "client agent" instances here.
     ::      uncommon pattern, very "soft". expect subscription failure and
     ::      handle it gracefully.
-    [base dap.bowl [%v4 base]]
-  ((safe-watch wire [ask dude] path) |)
+    [base dap.bowl [ver base]]
+  ((safe-watch wire [ship.nest dude] path) |)
```

Comment:
Oops, messed this up, you'll want `?(%v0 %v1 %v2)`, without `%v3` there.

But also, looking at this again: `ver` in place of `%v4` is probably bogus. For `%v3` we used `%v4` previously, because the subscription path shape (but not its response type) changed. We'll probably want `?+(ver ver ?(%v3 %v4) %v4)` or some such thing to make that match...

Reply:
I'll switch `ver` to `?+(ver ver ?(%v3 %v4) %v4)`, which compiles just fine. If you think we need to change that later though, feel free.